tests/installation/partitioning_raid.pm

```diff
@@ -343,10 +343,6 @@ sub run {
         send_key "right" unless is_storage_ng;
         assert_screen 'partitioning_raid-hard_disks-unfolded';
         send_key "down";
-        # CDROM is xvda and is seen as a disk block devide, we have to skip it
-        if (check_var('VIRSH_VMM_TYPE', 'linux')) {
```

Comment:
This worked on SLE15 because `xvda` item in the tree is unfolded by default and has two additional sub-options, but is not unfolded on SLE12SP4. I suggest to keep the code and run it only on SLE15+.

Reply:
That is true, maybe I am wrong, but anyway we will get to the expected disk using below code.
```
 my @devices = qw(vda vdb vdc vdd);
    @devices = qw(xvdb xvdc xvdd xvde) if check_var('VIRSH_VMM_FAMILY', 'xen');
    @devices = qw(sda sdb sdc sdd)     if check_var('VIRSH_VMM_FAMILY', 'hyperv');
    for (@devices) {
        send_key_until_needlematch "partitioning_raid-disk_$_-selected", "down";
        # storage-ng requires bios boot partition if not UEFI and not OFW
```

Reply:
I think you are right.